state/modeluser.go

```diff
@@ -170,85 +170,62 @@ func (st *State) removeModelUser(user names.UserTag) error {
 	return nil
 }
 
-// UserModel contains information about an model that a
-// user has access to.
-type UserModel struct {
-	*Model
-	User names.UserTag
-}
-
-// LastConnection returns the last time the user has connected to the
-// model.
-func (e *UserModel) LastConnection() (time.Time, error) {
-	db, dbCloser := e.modelDatabase()
-	defer dbCloser()
-	lastConnections, lastConnCloser := db.GetRawCollection(modelUserLastConnectionC)
-	defer lastConnCloser()
-
-	lastConnDoc := modelUserLastConnectionDoc{}
-	id := ensureModelUUID(e.ModelTag().Id(), strings.ToLower(e.User.Id()))
-	err := lastConnections.FindId(id).Select(bson.D{{"last-connection", 1}}).One(&lastConnDoc)
-	if (err != nil && err != mgo.ErrNotFound) || lastConnDoc.LastConnection.IsZero() {
-		return time.Time{}, errors.Trace(NeverConnectedError(e.User.Id()))
-	}
-
-	return lastConnDoc.LastConnection, nil
-}
-
-// allUserModels returns a list of all models with the user they belong to.
-func (st *State) allUserModels() ([]*UserModel, error) {
-	models, err := st.AllModels()
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	var result []*UserModel
-	for _, model := range models {
-		result = append(result, &UserModel{Model: model, User: model.Owner()})
-	}
-	return result, nil
-}
-
 // ModelsForUser returns a list of models that the user
```

Comment:
It really returns the UUIDs right?
SHould these methods that still say "Models" be renamed to "ModelUUIDs" for better clarity?